test/integration/channels/slack/setup.sh

```diff
@@ -56,10 +56,13 @@ ${WSK} package update ${PIPELINE_SEND_TEXT}_slack \
 
 ${WSK} action update ${PIPELINE_SEND_TEXT}_slack/receive ./channels/slack/receive/index.js -a web-export true > /dev/null
 ${WSK} action update ${PIPELINE_SEND_TEXT}_slack/post ./channels/slack/post/index.js > /dev/null
+${WSK} action update ${PIPELINE_SEND_TEXT}_slack/multiple_post ./channels/slack/multiple_post/index.js > /dev/null
```

Comment:
This looks fine but I'm forgetting why we decided to have different pipelines for the integration tests. Can you remind me? I think the artifacts(including the postsequence) should've been created at the time of test artifact setup so why do we need this? I'm confused.. 😕